src/main/webapp/jsp/instructorStudentRecords.jsp

```diff
@@ -10,7 +10,7 @@
   <script type="text/javascript" src="/js/instructorStudentRecords.js"></script>
 </c:set>
 <c:set var="bodyTitle">${fn:escapeXml(data.studentName)}'s Records<small class="muted"> - ${data.courseId}</small></c:set>
-<ti:instructorPage pageTitle="TEAMMATES - Instructor" jsIncludes="${jsIncludes}" bodyTitle="${bodyTitle}">
+<ti:instructorPage pageTitle="Student Records" jsIncludes="${jsIncludes}" bodyTitle="${bodyTitle}">
```

Comment:
Just highlighting that if we drop the `pageTitle` attribute, the title would include the student's name.